package.json

```diff
@@ -71,14 +72,14 @@
     "webpack-cli": "3.2.1"
   },
   "peerDependencies": {
+    "@edx/frontend-logging": "^3.0.1",
```

Comment:
If not using a new feature, David and I were discussing that these should be the lowest that is ok to not warn unnecessarily.  Can this remain 2.0.1?